backend/models/feedback_details.py

```diff
@@ -0,0 +1,14 @@
+from backend.models.feedback import Feedback
+
+__authors__ = ["Matt Vu"]
+__copyright__ = "Copyright 2023"
+__license__ = "MIT"
+
+
+class FeedbackDetails(Feedback):
```

Comment:
Since there are no back-populating relationship fields, is this model necessary?